.helm/starter/Chart.yaml

```diff
@@ -1,7 +1,9 @@
 ---
 apiVersion: v2
-appVersion: 0.1.0
-description: A Helm chart for Kubernetes
-name: starter
+description: The community-supported AWX-Operator Helm Chart
+name: awx-operator
 type: application
+# the current version of this helm chart
 version: 0.1.0
```

Comment:
Do we want to go ahead and change the chart version to CalVer or should we do that as a follow up?

Reply:
Feel free to change it, I'm not sure I understand where the current version starts but I think it would make sense to do it in this PR

Reply:
On second thought I'm a little unsure about switching this to CalVer in this PR. It probably feels somewhat arbitrary to change it now so I'm inclined to go with this version until we have some actual helm chart improvements.